jetty-common/src/main/java/org/jboss/arquillian/container/jetty/AbstractJettyEmbeddedConfiguration.java

```diff
@@ -282,5 +284,22 @@ public boolean isH2cEnabled() {
     public void setH2cEnabled(boolean h2cEnabled) {
         this.h2cEnabled = h2cEnabled;
     }
+
+    public Map<String, String> getHttpConfigurationProperties() {
+        return httpConfigurationProperties;
+    }
+
+    public void setHttpConfigurationProperties(String httpConfigurationProperties) {
+        this.httpConfigurationProperties = new HashMap<>();
+        String[] splittedLines = httpConfigurationProperties.split(" ");
```

Comment:
weird, and I think this is wrong.
some property values can contain spaces. (eg: format strings)
how can this possibly work?

Perhaps it should be proper CSV instead.

``` csv
"jetty.start.dumpAfterStart","true"
"jetty.requestlog.formatString", "%{client}a - %u %{dd/MMM/yyyy:HH:mm:ss ZZZ|GMT}t"
"jetty.requestlog.dir", "C:\path with spaces\logs"
```

Reply:
Another, probably simpler approach.
Just use a normal properties file.
And have the httpConfigurationProperties point to that file.

Reply:
Or you can have the XML for httpConfigurationProperties just BE a properties file in contents (some line trimming might be needed, it being XML and all)
And just parse it with `Properties.load(Reader)`

Like this ...

``` java
package props;

import java.io.IOException;
import java.io.StringReader;
import java.util.Collections;
import java.util.Objects;
import java.util.Properties;

public class RawPropertiesLoadDemo
{
    public static void main(String[] args) throws IOException
    {
        String rawProps = """
            key=value
            foo=property with spaces
            bar="Property with quotes"
            """;

        Properties props = new Properties();
        try (StringReader reader = new StringReader(rawProps))
        {
            props.load(reader);
        }
        Collections.list(props.propertyNames())
            .stream()
            .map(Objects::toString)
            .sorted()
            .forEach((name) ->
                System.out.printf("Prop [%s] = [%s]%n", name, props.getProperty(name)));
    }
}
```

which results in ...

```
Prop [bar] = ["Property with quotes"]
Prop [foo] = [property with spaces]
Prop [key] = [value]
```

Reply:
Looks like Properties.load(Reader) trims whitespace anyway (before and after each line), so this just became even easier.

Reply:
this is using similar code as already done for `mimeTypes` and `inferredEncodings`
because the main goal is to configure properties that are `boolean` or `int` or very simple `String` (HttpConfiguration)
but the raw loading sounds easier

Reply:
mime types and inferred encoding is far more strict and cannot have spaces in odd places like arbitrary properties can.

Reply:
fixed.